packages/blockprotocol/cli.js

```diff
@@ -1,20 +1,10 @@
 #!/usr/bin/env node
 
-import chalk from "chalk";
 import commandLineArgs from "command-line-args";
 import commandLineUsage from "command-line-usage";
 
 import { commandLookup } from "./cli/commands.js";
-import { run as printHelp } from "./cli/commands/help.js";
-import { printSpacer } from "./cli/print-spacer.js";
-
-/**
- * @param {string} errorMessage
- */
-const printErrorMessage = (errorMessage) => {
-  console.log(chalk.red(`${errorMessage}. Please check usage.`));
-  printSpacer();
-};
+import { printErrorMessage } from "./cli/print-error-message.js";
```

Comment:
This fn was extracted to `cli/print-error-message.js` to be able to move `print-spacer` into `cli/shared` for clarity.